client/app/styles/index.styl

```diff
@@ -179,3 +293,56 @@ aside
                 height      6em
                 width       100%
                 margin      2em 0em
+
+// ONBOARDING NOTIFICATION
+notification-width = (546/base-font-size)em
+.coz-notifier
+    z-index 100
+
+.coz-notification
+    display block
+    position absolute
+
+    top (16/base-font-size)em
+
+    left 0
+    margin 0 (16/base-font-size)em
+
+    border-radius (10/base-font-size)em
+    background-color #2bba40
+    box-shadow 0 (6/base-font-size)em (18/base-font-size)em 0 rgba(50, 54, 63, 0.23)
+
+    box-sizing border-box
+    padding 1em
+
+    color white
+
+    // When only a title is displayed
+    min-height: (96/base-font-size)em
+    padding-left (96/base-font-size)em
+    background-image url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='48' height='48' viewBox='0 0 48 48'><path fill='#FFFFFF' fill-rule='evenodd' d='M24,48 C37.254834,48 48,37.254834 48,24 C48,10.745166 37.254834,0 24,0 C10.745166,0 0,10.745166 0,24 C0,37.254834 10.745166,48 24,48 Z M13.4142136,24.5857864 C12.633165,23.8047379 11.366835,23.8047379 10.5857864,24.5857864 C9.80473785,25.366835 9.80473785,26.633165 10.5857864,27.4142136 L17.5857864,34.4142136 C18.366835,35.1952621 19.633165,35.1952621 20.4142136,34.4142136 L35.4142136,19.4142136 C36.1952621,18.633165 36.1952621,17.366835 35.4142136,16.5857864 C34.633165,15.8047379 33.366835,15.8047379 32.5857864,16.5857864 L19,30.1715729 L13.4142136,24.5857864 Z'/></svg>")
```

Comment:
I definitely enforce the use of SVG embed in stylesheet, especially for icons (and I'll be a nazi on this, you're now aware :)).

Let's say your on a mobile connectivity, you CSS is loaded but the ruleset containing the external SVG isn't applied at this time because it concerns a notification that'll pop later. Then suddenly (like our [Francis](http://www.francisblaireau.com/) friend) you connection drop, and your notification pop. You now face an element without its icon because this one can't be retrieved as there's not connectivity at all. It sucks. A lot. Really.

With embeded SVG icons, when your CSS is applied it means the stylesheet is fully loaded. So icons are available and even if your connection drop (Hop!), you'll never loose the Ui.

But I agree embedding SVG in stylesheet is a bit messy, this is why stylus provides us the magnificent [`embedurl`](http://stylus-lang.com/docs/functions.url.html#utf8-encoding-for-svgs) directive, so please use it to embed external source SVG files in the final CSS. I'll bring you kisses.

Thanks :)

Your devoted Ui Nazi